sklearn/svm/tests/test_svm.py

```diff
@@ -985,6 +985,31 @@ def test_ovr_decision_function():
     assert np.all(pred_class_deci_val[:, 0] < pred_class_deci_val[:, 1])
 
 
+@pytest.mark.parametrize("svc", [svm.SVC, svm.NuSVC])
```

Comment:
Total nitpick, but here "svc" is a class. I think that it should be named with capitals, eg "SVCClass"